app/views/idv/in_person/address.html.erb

```diff
@@ -1,15 +1,89 @@
 <% title t('titles.doc_auth.verify') %>
 
-<h1 class="h3 margin-y-0">
-  <%= t('in_person_proofing.headings.address') %>
-</h1>
-
-<%= validated_form_for :doc_auth,
-                       url: url_for,
-                       method: 'put',
-                       html: { autocomplete: 'off', class: 'margin-y-5' } do |f| %>
-  <%= f.button :button,
-               t('doc_auth.buttons.continue'),
-               type: :submit,
-               class: 'usa-button--big usa-button--wide' %>
-<% end %>
+<%= render PageHeadingComponent.new.with_content(t('in_person_proofing.headings.address')) %>
+
+<p>
+  <%= t('in_person_proofing.body.address.info') %>
+  <%= new_window_link_to(t('in_person_proofing.body.address.learn_more'), MarketingSite.security_and_privacy_practices_url) %>
+</p>
+
+<div class="margin-top-4 margin-bottom-4">
+  <%= validated_form_for(
+        :doc_auth, url: url_for, method: 'PUT',
+                   html: { autocomplete: 'off', class: 'margin-top-2' }
+      ) do |f| %>
+    <div class="margin-bottom-3">
```

Comment:
These `<div>` wrappers currently aren't doing anything, since each field will apply a `margin-bottom-4` which will take precedence over this. Can we remove them?